pages/hosted_private_cloud/sap_on_ovhcloud/cookbook_sap_logs_on_ovhcloud_logs_data_platform_analyze_and_work_with_your_logs/guide.de-de.md

```diff
@@ -1,13 +1,128 @@
 ---
 title: "SAP logs on OVHcloud Logs Data Platform - Analyze and work with your logs"
-excerpt: "This guide is designed to assist you in retrieving and exploiting your SAP logs on the OVHcloud Logs Data Platform service"
-updated: 2024-03-30
-flag: hidden
+excerpt: "This guide is designed to assist you in retrieving and exploiting your SAP logs on the OVHcloud Logs Data Platform feature"
+updated: 2024-04-24
 ---
 
+<style>
+details>summary {
+    color:rgb(33, 153, 232) !important;
+    cursor: pointer;
+}
+details>summary::before {
+    content:'\25B6';
+    padding-right:1ch;
+}
+details[open]>summary::before {
+    content:'\25BC';
+}
+</style>
+
 ## Objective
 
+This guide is designed to assist you in retrieving and exploiting your SAP logs on the OVHcloud Logs Data Platform feature.
+
+If you haven't configured the SAP logs on OVHcloud Logs Data Platform feature, please use our guide [SAP logs on OVHcloud Logs Data Platform - Solution Setup](/pages/hosted_private_cloud/sap_on_ovhcloud/cookbook_sap_logs_on_ovhcloud_logs_data_platform_solution_setup) to get started.
+
+## Requirements
+
+- [SAP logs on OVHcloud Logs Data Platform configured](/pages/hosted_private_cloud/sap_on_ovhcloud/cookbook_sap_logs_on_ovhcloud_logs_data_platform_solution_setup)
+
+## Instructions
+
+### View your logs
+
+With the implementation of SAP logs on OVHcloud Logs Data Platform feature, a collection of logs is accessible in your data stream.
+
+To visualize your logs, you have the option to use Graylog or OpenSearch Dashboards. OpenSearch Dashboards allows you to create customized dashboards and searches that can help you to response to your different use cases.
+
+OVHcloud manages a list of SAP files which are pre-configured and can be directly used. You can access to these SAP files on [GitHub repository](https://github.com/ovh/sap-logs-on-ovhcloud-logs-data-platform).
+
+### Filter your logs
+
+Several pre-defined fields have been created in order to make your navigation through your logs easier.
+
+Below, a summary of pre-defined fields:
+
+| Field name  | Description |
+| ----------  | ----------- |
+| filename    | Refers to the location of the file that has been tracked. |
+| hana_sid    | HANA SID of your SAP HANA system or the reference of the HANA SID used by your SAP system. |
+| hana_tenant | SAP HANA tenant in a multi-tenanted environment. |
+| message     | Raw log. |
+| sap_sip     | SAP SID of your SAP system or the reference of SAP SID used for this SAP HANA database. |
+| saplog      | Defines the category of your log.<br>Examples: syslog, security_audit, dev_w, nameserver, etc. |
+| source      | Hostname of the server which forwarded the log. |
+| timestamp   | Timestamp (UTC). |
+
+Examples of filter usage:
+
+<details>
+<summary>Logs of the work process trace files (dev_w*)</summary>
+
+![filter_dev_w1](images/filter_dev_w1.png){.thumbnail}
```

Comment:
These images won't show. The `<details>`is HTML so it implies everything in it is in HTML too.
If you want to keep this collapsible display, you need to use HTML such as `<img src="/images/imagename.png" alt="alt-text"/>`